test/python/dialects/indexing/dialect.py

```diff
@@ -225,16 +225,211 @@ def testConcatenateOp():
   i32 = IntegerType.get_signless(32)
   with mlir_mod_ctx() as module:
 
-    @func.FuncOp.from_py_func()
```

Comment:
(Not overly important but the changes to this function could have been a different commit/PR... 😛)